data-serving/scripts/convert-data/constants.py

```diff
@@ -54,18 +54,24 @@
     ('bergamo', 'lombardia', 'italy'): ('Bergamo', 'Lombardia', 'Italy'),
     ('harbin',): ('Harbin City', 'Heilongjiang', 'China'),
     ('lombardy', 'italy'): (None, 'Lombardia', 'Italy'),
-    ('lombardy italy',): (None, 'Lombardia', 'Italy'),
     ('london',): ('London', 'England', 'United Kingdom'),
     ('madrid',): (None, 'Madrid', 'Spain'),
+    ('madrid', 'spain'): (None, 'Madrid', 'Spain'),
+    ('miami', 'florida', 'united states'): ('Miami-Dade County', 'Florida', 'United States'),
     ('milan',): ('Milan', 'Lombardia', 'Italy'),
     ('milan', 'italy'): ('Milan', 'Lombardia', 'Italy'),
     ('nanning',): ('Nanning City', 'Guangxi', 'China'),
+    ('naples',): ('Naples', 'Campania', 'Italy'),
+    ('rio de janeiro', 'brazil'): ('Rio de Janeiro', 'Rio de Janeiro', 'Brazil'),
     ('sao paulo',): (None, 'Sao Paulo', 'Brazil'),
-    ('south tyrol',): ('Trentino-Alto Adige', 'Italy'),
-    ('south tyrol', 'italy'): ('Trentino-Alto Adige', 'Italy'),
+    ('south tyrol',): (None, 'Trentino-Alto Adige', 'Italy'),
+    ('south tyrol', 'italy'): (None, 'Trentino-Alto Adige', 'Italy'),
+    ('taiwan',): (None, 'Taiwan', None),
```

Comment:
Now that I see this row, I'm guessing there's some sort of conditional flexibility on how these tuples are interpreted?

Reply:
a lot of these are to disambugate when they match multiple admin regions, ex. madrid is the city and the county.